pymc3/tests/test_distributions.py

```diff
@@ -3000,3 +2998,22 @@ def test_distinct_rvs():
         pp_samples_2 = pm.sample_prior_predictive(samples=2)
 
     assert np.array_equal(pp_samples["y"], pp_samples_2["y"])
+
+
+def test_lkjcholeskycov():
+    D = 15
+    samples = 5
+    dist_shape = ((D * (D + 1)) // 2,)
+
+    with pm.Model() as model:
+        sd_dist = pm.HalfCauchy.dist(beta=2.5, size=(D))
```

Comment:
One way to avoid the `sd_dist` problem is to ensure that the output shape of `dist` over here is (`n`) i.e. the number of random draws required in `LKJCholeskyCov` so that they automatically get evaluated as required.